Replace static diagram with maintained diagrams

sldblog opened the pull request:
## What does this pull request do?

Replaces the static diagram with maintained diagrams.

The current diagram is great, but already out of date due to `cla_public` and `fala` no longer running on AWS.

The aim of these changes is to replace the diagram with the ones in the architecture documentation repo, which is maintained and long-lived.

**Before**: https://github.com/ministryofjustice/laa-cla-e2e-tests/blob/062bb844f599c5447bdd6d2b1c19adec1d2c84b5/README.md
**After**: https://github.com/ministryofjustice/laa-cla-e2e-tests/blob/4d8ffa1cd0fd4ee4debcc4d0c2c0a012ee5db1c7/README.md

## Any other changes that would benefit highlighting?

I think the currently live diagram is **better** than these detailed ones for the purpose of this readme, but I also _really_ dislike updating these purpose-built diagrams.

💭 Maybe this points towards that we should treat these apps as a single system?

*Bonus*: Fixes spelling inconsistencies between "Civil Legal Aid" and "Civil Legal Advice", "Advisor" and "Adviser".

sldblog committed:
Replace static diagram with a maintained diagram

The current diagram is great, but already out of date due to
`cla_public` and `fala` no longer running on AWS.

The aim of these changes is to replace the diagram with the ones in the
architecture documentation repo, which is maintained and long-lived.

*Bonus*: Fixes inconsistencies between "Civil Legal Aid" and "Civil
Legal Advice", "Advisor" and "Adviser".

said-moj commented:
images in the after readme don't work  https://github.com/ministryofjustice/laa-cla-e2e-tests/blob/aa9d1122b4585655699418c75ba29d4403416288/README.md

sldblog commented:
TIL media links do not redirect when the repository is renamed :(

sldblog committed:
Update media links to use new repository name

The architecture documentation repo was recently renamed to
`laa-architectural-diagrams` and it seems media URLs do not
automatically redirect :sob:

sldblog commented:
@said-moj Fixed in 4d8ffa1cd0fd4ee4debcc4d0c2c0a012ee5db1c7

sldblog commented:
🎩 🙇
